### ISSUE TITLE
Computing filename when setting org-projectile-per-project-filepath

### ISSUE THREAD

**Issue by jmignault** [colonelpanic8/org-project-capture#41]
What I want is to have a separate org file for each project to-do list and have all of them reside in ```org-directory```, so I am setting ```org-projectile-per-project-filepath``` to this function:

```
(defun org-projectile-orgfile-path ()
  (concat org-directory "/" (projectile-project-name) "_todo.org"))
```

 But then org-projectile-todo-files contains entries like
```~/Documents/workspace/hexagram//Users/jmignault/Dropbox/org/dotfiles_todo.org```

when what I want is
```/Users/jmignault/Dropbox/org/hexagram_todo.org```

Am I missing something? Do i need to put this in a hook somewhere rather than running this at startup?

**Comment by IvanMalison** [comment]
@jmignault I'm on mobile right now so I can't confirm this, but I think `(projectile-project-name)` gives you the path to the project. It seems like you probably just want to apply `directory-file-name` to get the name of the project

**Comment by jmignault** [comment]
Actually it just gives you the bare name of the project. Am I correct in assuming that if you run ```(org-projectile-per-project)``` that the only directory in which the per-project org file can reside is the project directory? ```org-projectile-per-project-filepath``` just constructs the org-file filename, correct, not the complete path to the file?

**Comment by IvanMalison** [comment]
@jmignault Yes, as currently formulated, the per project strategy does a concat of the project filepath with `org-projectile-per-project-filepath`.

You could specify additional peices of path in `org-projectile-per-project-filepath`, but it will alway be interpreted relative to the projectile path. It sounds like you want to store the file somewhere else completely, correct?

Doing this is certainly possible, but you have to write a bit of elisp. Basically you want to copy most of the  per project code (starts https://github.com/ivanmalison/org-projectile/blob/53c193a209d663b33de71fe7d35eb40c0d0e15b6/org-projectile.el#L137), but replace the
`org-projectile-get-project-todo-file` (and all of the invocations) with something like

```
(defun org-projectile-get-project-todo-file-from-dropbox (project-path)
  (concat "~/Dropbox/" (file-name-nondirectory (directory-file-name project-path)) ".org"))
```

In fact, I think you could simply overwrite the definition of `org-projectile-get-project-todo-file` with that and things should work.

Hope that helps

**Comment by colonelpanic8** [comment]
This was fixed by e474dca2b52c63775542a95afbebb9306ad7bede / #42
